dist/htmlminifier.js

```diff
@@ -285,8 +237,30 @@
       parseEndTag();
     }
 
-    function parseStartTag( tag, tagName, rest, unary ) {
-      var unarySlash = false;
+    function parseStartTag(input) {
+      var start = input.match(startTagOpen);
+      if (start) {
+        var match = {
```

Comment:
I'd have no idea on this one. I can only say I haven't used it in any of my projects yet.